### DISCUSSION TITLE
PvP implementation

### DISCUSSION THREAD

**Discussion by rekterakathom** [ArmaOverthrow/Overthrow.Arma4#25] [Ideas]
## General discussion about if/how PvP might be implemented

My initial suggestion: Allow players to join the occupying faction when they join a server. This option is only available if the current amount of occupying faction players is less than the amount of resistance players, for starters let's say that occupying faction gets a "slot" for every three slots that get occupied in resistance.

Server admins would be able to change the occupying faction player limits, whitelist/blacklist players, or disable the system entirely.

Occupying faction players would get basic equipment and be tasked with fighting the resistance (duh). Progression for these players might get added later on, initially this would do.

### Some issues to address
- Lack of direction for OF players
- Possible griefing or metagaming (OF players give stuff to resistance)
- Fragmentation of playerbase (shifts players away from the resistance gameplay)
- Potentially increased code complexity because player side won't be constant

**Comment by aaronstatic** [comment]
The initial suggestion is basically what I had planned eventually, yes.. I like the idea of limiiting the slots to 1/3 and making it configurable. I'd also add the following:

- Default pvp to off. Server admins need to specifically turn it on
- Once a player has joined either side, they are limited only to that side without admin involvement on that server. To avoid people cheesing by joining either side to find positions etc
